Example/DisplaySwitcher/ViewControllers/UserViewController/UserViewController.swift

```diff
@@ -20,6 +20,7 @@ class UserViewController: UIViewController {
     @IBOutlet private weak var searchBar: UISearchBar!
     @IBOutlet private weak var rotationButton: RotationButton!
     
+    private var tap:UITapGestureRecognizer!
```

Comment:
@serejahh Did you mean that I have to remove spaces in lines 121-127 and add space after colon in line 23 then PR ?